state/model_test.go

```diff
@@ -498,6 +499,26 @@ func (s *ModelSuite) TestAllUnits(c *gc.C) {
 	})
 }
 
+func (s *ModelSuite) TestAllEndpointBindings(c *gc.C) {
+	s.Factory.MakeSpace(c, &factory.SpaceParams{
+		Name: "one", ProviderID: network.Id("provider"), IsPublic: true})
+	state.AddTestingApplicationWithBindings(
+		c, s.State, "wordpress", state.AddTestingCharm(c, s.State, "wordpress"),
+		map[string]string{"db": "one"})
+
+	model, err := s.State.Model()
+	c.Assert(err, jc.ErrorIsNil)
+
+	listBindings, err := model.AllEndpointBindings()
+	c.Assert(listBindings, gc.HasLen, 1)
+
+	for _, bind := range listBindings {
```

Comment:
No need for for loop. We know there's only one, do this:

c.Assert(listBindings[0], jc.DeepEquals, ApplicationEndpointBindings{
    AppName: "wordpress",
    Bindings: map[string]string{"db": "one"},
})